BUILDING.md

```diff
@@ -0,0 +1,48 @@
+# How to Compile
+
+FileOnQ.Imaging.Raw uses a mix of .NET Framework, .NET, and native C/C++ projects. You will need to have additional build tools installed to properly build this locally. This readme serves as a guide for new developers to contribute.
+
+## Visual Studio and .NET
+
+This project uses a combination of Visual Studio and .NET CLI tools for building and developing.
+
+* Visual Studio 2019 (16.10.3) or newer
+  * Workload: Desktop development with C++
+  * Component: Windows 10 SDK (10.0.18)
+* .NET CLI 5.0.300 or newer
+
+If you are using an older version of Visual Studio or .NET CLI you won't have access to the `_OR_GREATER` preprocessor directives. This will result in runtime issues when running the library locally.
+
+## Build Dependencies
+
+Grab the LibRaw dependencies by running the command `git submodule update --init --recursive` in the repo's root folder. You will also need to download and install [CUDA SDK 11.4](https://developer.nvidia.com/cuda-11-4-3-download-archive)
```

Comment:
Let's try something like this:
--
* Visual Studio 2019 for C++ Development
* [CUDA SDK 11.4](https://developer.nvidia.com/cuda-11-4-3-download-archive)

Clone the project with submodules to include LibRaw dependency.
```
git clone --recurse-submodules https://github.com/FileOnQ.Imaging.Raw.git
```